lib/tasks/migrate_role_content.rake

```diff
@@ -0,0 +1,16 @@
+namespace :data_hygiene do
```

Comment:
I think this can go in the existing file for `data_hygiene` tasks as it's relatively small and the file itself is currently not very long: https://github.com/alphagov/whitehall/blob/master/lib/tasks/data_hygiene.rake#L10